.pre-commit-config.yaml

```diff
@@ -43,11 +43,14 @@ repos:
     -   id: mdformat
         additional_dependencies:
         -   mdformat-gfm
--   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.5.1
+-   repo: local
     hooks:
-    -   id: mypy
-        exclude: tests/core
```

Comment:
Do we still want to exclude tests?

Reply:
It switches the command from just running `mypy` on all the files to only running it on files within the `eth_utils` package. That's the `mypy -p` flag. So don't need to exclude explicitly.

Reply:
That sounds good! A test in `tests/mypy` for abi types was just added in my PR. Just to be sure, I'd add the dir to the command as well?

Reply:
Sorry, yeah, I hadn't noticed that. I'll add it.